lib/lightning_web/live/run_live/index.ex

```diff
@@ -41,7 +41,8 @@ defmodule LightningWeb.RunLive.Index do
     cancelled: :boolean,
     killed: :boolean,
     exception: :boolean,
-    lost: :boolean
+    lost: :boolean,
+    rejected: :boolean
```

Comment:
this is a separate fix — the filters weren't working for "rejected" and this does the trick!